pypandora/api.py

```diff
@@ -93,3 +93,17 @@ def task_status(self, task_id: str, seed: Optional[str]=None) -> Dict[str, Any]:
         r = self.session.get(url, params={'task_id': task_id, 'seed': seed})
         to_return = r.json()
         return to_return
+
+    def worker_status(self, task_id: str, all_workers: int, details : int, seed: Optional[str]=None, worker_name: Optional[str]=None) -> Dict[str, Any]:
```

Comment:
I like having sane defaults, and staying as pythonesque as possible, so the header of the method should be:
`def worker_status(self, task_id: str, all_workers: bool=True, details: bool=False, seed: Optional[str]=None, worker_name: Optional[str]=None) -> Dict[str, Any]:`

Advantages:
* has sane default, the only required parameter is task_id, everything else is optional
* flask expects integers, transform `bool` to `int` when you do the get request: in our case the only relevant values are True/False, 0/1 is confusing